llama-index-integrations/readers/llama-index-readers-pdf-marker/README.md

```diff
@@ -0,0 +1,28 @@
+# LlamaIndex Readers Integration: Pdf-Marker
+
+Uses the [pdf-marker](https://github.com/VikParuchuri/marker/) library to extract the content of a PDF file.
+
+From the original README:
+
+Marker converts PDF to markdown quickly and accurately.
+
+- Supports a wide range of documents (optimized for books and scientific papers)
+- Supports all languages
+- Removes headers/footers/other artifacts
+- Formats tables and code blocks
+- Extracts and saves images along with the markdown
+- Converts most equations to latex
+- Works on GPU, CPU, or MPS
+
+## Usage
+
+Here's an example usage of the PDFMarkerReader.
+
+```python
+from llama_index.readers.pdf_marker import PDFMarkerReader
+from pathlib import Path
+
+path = Path("/path/to/pdf")
+reader = PDFMarkerReader()
+reader.load_data(path)
+```
```

Comment:
kindly consider : 
- adding the Vik's DataLab Surya Licence for release
- Mentioning it in the README.md
- Adding a clear explanation of the licence terms in one or two sentences in the README.md

Reply:
LGTM 🎉

Reply:
I'll also update the package license and add a LICENSE.md to this folder